src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -486,6 +487,11 @@ private async Task<CreationResultStatus> ExecuteAsync()
                     return updateCheckResult ? CreationResultStatus.Success : CreationResultStatus.CreateFailed;
                 }
 
+                if (_commandInput.SearchOnline)
+                {
+                    return await CliTemplateSearchCoordinator.SearchForTemplateMatchesAsync(EnvironmentSettings, _commandInput, _defaultLanguage);
```

Comment:
Do we manage a synchronization context in templating engine or can we call `.ConfigureAwait(false)` after async calls?